services/harmony/app/frontends/service-image-tags.ts

```diff
@@ -153,7 +155,25 @@ export async function updateServiceImageTag(
 
   const { tag } = req.body;
 
-  // TODO HARMONY-1701 run deployment script here
+  const currentPath = __dirname;
+  const cicdDir = path.join(currentPath, '../../../../../harmony-ci-cd');
+  // Set the current working directory to the cicd directory
+  process.chdir(cicdDir);
```

Comment:
I believe this is causing test breakages and also putting test files in my harmony-ci-cd directory. We need to find a way to call the script without changing the current directory for the harmony application.

Reply:
Good point. I changed the code to pass the script directory as the cwd options of exec call.